src/main/kotlin/com/amazon/ion/plugin/intellij/formatting/IonLanguageCodeStyleSettingsProvider.kt

```diff
@@ -23,6 +23,12 @@ class IonLanguageCodeStyleSettingsProvider : LanguageCodeStyleSettingsProvider()
     }
 
     override fun customizeDefaults(commonSettings: CommonCodeStyleSettings, indentOptions: IndentOptions) {
+        // When using spaces or tabs: Use two spaces for indents.
+        indentOptions.INDENT_SIZE = 2
+
+        // When using tabs: Treat a tab as two spaces.
+        indentOptions.TAB_SIZE = 2
```

Comment:
Is there a way to default-to replacing tabs with spaces?

Reply:
It looks like it. [CommonCodeStyleSettings.IndentOptions](https://github.com/JetBrains/intellij-community/blob/fac93f2bcf61fc41717b2133e25af76e95cfe528/platform/code-style-api/src/com/intellij/psi/codeStyle/CommonCodeStyleSettings.java#L1008-L1029) has all the details. I don't see anything else worth setting, but I'll update for spaces instead of tabs as a default